docs/source/notebooks/elements.ipynb

```diff
@@ -107,6 +107,7 @@
     "                                  connection=connection)\n",
     "globals().update(**objects)\n",
     "\n",
+    "data['models']['simulation']['npoint'] = 1000000\n",
```

Comment:
What was the previous value?

Reply:
Addressed via https://github.com/NSLS-II/sirepo-bluesky/pull/88/commits/6ff9be1ddac79406e159239cbf244be6feb51374.